homework/index.js

```diff
@@ -29,22 +33,123 @@
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+
+  function addTableRow(table, header, value) {
+    const tr = createAndAppend('tr', table, { class: 'tr' });
+    createAndAppend('th', tr, { text: header, class: 'keys' });
+    createAndAppend('td', tr, { text: value, class: 'values' });
```

Comment:
I also added some other classes and didn't use them like values class in case of editing in the future. If my this habit is mistaken, I can abandon it.